packages/web-components/package.json

```diff
@@ -94,8 +94,8 @@
   "dependencies": {
     "@microsoft/fast-colors": "^5.1.0",
     "@microsoft/fast-components-styles-msft": "^4.29.0",
-    "@microsoft/fast-element": "^0.15.1",
-    "@microsoft/fast-foundation": "^1.5.0",
+    "@microsoft/fast-element": "^0.15.2",
+    "@microsoft/fast-foundation": "^1.6.0",
```

Comment:
@khamudom did you happen to see yarn.lock changes locally which weren't checked in?

Reply:
no, it didn't make any changes to the yarn.lock file

Reply:
ok, I see one now. it updated when I ran yarn at the root.